### DISCUSSION TITLE
How to get which message was edited in a SkypeEditMessageEvent?

### DISCUSSION THREAD

**Discussion by truenicoco** [Terrance/SkPy#212] [Q&A] [answered]
Hi Terrance,

When a contact edits a message, skpy fires a `SkypeEditMessageEvent` that has a new `msgId`. I feel a little dumb but I don't see where I can find the original message ID in this event. Could you help me understand how I can find it?

**Comment by Terrance** [comment, marked as answer]
You can match them up using [`SkypeMsg.clientId`](https://skpy.t.allofti.me/reference/api.html#skpy.msg.SkypeMsg.clientId) -- the main `id` will be different for each message and edit, but `clientId` should persist across edits (this is the same ID passed to [`SkypeChat.sendMsg(edit=...)`](https://skpy.t.allofti.me/reference/api.html#skpy.chat.SkypeChat.sendMsg) when updating your own messages).

**Reply by truenicoco** [reply]
Thanks a lot Terrance! I had figured out half of it but forgot to update here.
(I understood how to edit but I thought I needed to re-fetch the message before that, I did not see the `sendMsg()` `edit=` argument).
